setup.py

```diff
@@ -42,4 +42,9 @@
         'Programming Language :: Python :: 3 :: Only',
         'Programming Language :: Python :: Implementation :: PyPy',
     ],
+    extras_require={
+        'test': ['testfixtures>=1.2.0',
+                 'pytest>=3.4.0',
+                 ],
```

Comment:
Nit: I'd prefer to format this similar to other lines
```suggestion
        'test': [
          'testfixtures>=1.2.0',
          'pytest>=3.4.0',
        ],
```